contributing_to_docs/contributing.adoc

```diff
@@ -5,6 +5,7 @@
 :toc-title:
 :toclevels: 1
 :description: Basic information about the OpenShift GitHub repository
+:imagesdir: ../images
```

Comment:
`../images` is specified here because the image does not appear if `images` is stated. This is probably something to do with the symlink.